tests/test_crash_similarity.py

```diff
@@ -62,12 +61,11 @@ def test_order_similarity(self):
 
     def test_clean_func(self):
```

Comment:
We should move this and the following test to `test_utils.py`, since the functions were moved to utils.